cartesian_trajectory_controller/test/setup.launch

```diff
@@ -0,0 +1,42 @@
+<launch>
```

Comment:
ist might be good to use a plain `.xml` suffix. Otherwise, this will popup as suggestion when doing `roslaunch cartesian_trajectory_controller <tab>`

Reply:
Agreed, see b069093